x-team/auto-ui
# Contact autofocus on search input

**Release Type:** *Non-breaking feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2333

## Description

Includes autoFocus prop on SelectBox to auto-focus on search field.

## Checklist

- [x] if any snapshots have been changed, verify that component still works and looks as expected and update changed snapshot

## Related PRs


branch | PR
------ | ------
`XP-2333-contact-autofocus-on-search-input` | [Auto](https://github.com/x-team/auto/pull/1582)


## Steps to Test or Reproduce

> There's a new use case included for this case, it can be seen at http://localhost:6060/#!/SelectBox/11, the autoFocus will work loading that page, otherwise please test the feature on XP app instead Styleguidist, as described bellow:

1. Click on template selector icon at Contact form
1. Observe that the focus goes automatically to the search field


## Screenshots

![screen recording 2018-10-10 at 10 30 am](https://user-images.githubusercontent.com/131859/46740124-7243fc00-cc78-11e8-81c2-72be3d309f61.gif)
